presentation/src/main/java/me/androidbox/presentation/task/screen/TaskDetailScreen.kt

```diff
@@ -0,0 +1,184 @@
+
+            Column (modifier = modifier
+                .fillMaxWidth()
+                .background(
+                    color = White,
+                    shape = RoundedCornerShape(topStart = 32.dp, topEnd = 32.dp)
+                )
+                .padding(horizontal = 16.dp)
+            ) {
+                Spacer(modifier = modifier.height(30.dp))
+
+                AgendaHeader(
+                    agendaHeaderItem = AgendaHeaderItem.TASK,
+                    subTitle = taskDetailScreenState.taskTitle,
+                    description = taskDetailScreenState.taskDescription,
+                    isEditMode = taskDetailScreenState.isEditMode,
+                    onEditTitleClicked = { newTitle ->
+                        onEditTitleClicked(newTitle)
+                    },
+                    onEditDescriptionClicked = { newDescription ->
+                        onEditDescriptionClicked(newDescription)
+                    }
+                )
+
+                Spacer(modifier = modifier.height(26.dp))
+
+                AgendaDurationSimple(
+                    isEditMode = taskDetailScreenState.isEditMode,
+                    startTime = taskDetailScreenState.from.formatDateTime(TIME_PATTERN),
+                    startDate = taskDetailScreenState.from.formatDateTime(DATE_PATTERN),
+                    onStartDurationClicked = {
+                        calendarStateTimeDate.show()
+                    })
+
+                AgendaDropDownMenu(
+                    modifier = Modifier
+                        .background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor),
+                    shouldOpenDropdown = taskDetailScreenState.shouldOpenDropdown,
+                    onCloseDropdown = {
+                        taskDetailScreenEvent(TaskDetailScreenEvent.OnShowAlarmReminderDropdown(shouldOpen = false))
+                    },
+                    listOfMenuItemId = AlarmReminderItem.values().map { alarmReminderItem ->
+                        alarmReminderItem.stringResId
+                    },
+                    onSelectedOption = { item ->
+                        taskDetailScreenEvent(TaskDetailScreenEvent.OnAlarmReminderChanged(AlarmReminderItem.values()[item]))
+                        taskDetailScreenEvent(TaskDetailScreenEvent.OnShowAlarmReminderDropdown(shouldOpen = false))
+                    }
+                )
+
+                AlarmReminder(
+                    reminderText = stringResource(id = taskDetailScreenState.alarmReminderItem.stringResId),
+                    isEditMode = taskDetailScreenState.isEditMode,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .background(color = MaterialTheme.colorScheme.backgroundWhiteColor),
+                    onReminderClicked = {
+
+                    }
+                )
+                Spacer(modifier = modifier.height(26.dp))
+            }
+        }
+    }
+}
```

Comment:
@philipplackner @MKiperszmid For this screen if you look at the screenshot in the description it contains some black background. As there isn't enough content in the task screen to expand to the full height I am left with this blank space. Is there anything I can do to have the content to the maximum height to fill that blank space?

Reply:
Add a fillMaxSize to the column maybe?

Reply:
@MKiperszmid I did add the fillMaxSize but this didn't help. The outer column contains the black background.